pkg/services/meta/container.go

```diff
@@ -0,0 +1,129 @@
+package meta
+
+import (
+	"crypto/sha256"
+	"errors"
+	"fmt"
+	"maps"
+	"os"
+	"path"
+	"sync"
+
+	"github.com/nspcc-dev/neo-go/pkg/core/mpt"
+	"github.com/nspcc-dev/neo-go/pkg/core/storage"
+	"github.com/nspcc-dev/neo-go/pkg/core/storage/dbconfig"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+	cid "github.com/nspcc-dev/neofs-sdk-go/container/id"
+	objectsdk "github.com/nspcc-dev/neofs-sdk-go/object"
+)
+
+type containerStorage struct {
+	m        sync.RWMutex
+	opsBatch map[string][]byte
+
+	path string
+	mpt  *mpt.Trie
+	db   storage.Store
+}
+
+func (s *containerStorage) drop() error {
+	s.m.Lock()
+	defer s.m.Unlock()
+
+	err := s.db.Close()
+	if err != nil {
+		return fmt.Errorf("close container storage: %w", err)
+	}
+
+	err = os.RemoveAll(s.path)
+	if err != nil {
+		return fmt.Errorf("remove container storage: %w", err)
+	}
+
+	return nil
+}
+
+func (s *containerStorage) putObject(e objEvent) error {
+	s.m.Lock()
+	defer s.m.Unlock()
+
+	newKVs := make(map[string][]byte)
+	commsuffix := e.oID[:]
+
+	// batching that is implemented for MPT ignores key's first byte
+
+	newKVs[string(append([]byte{0, oidIndex}, commsuffix...))] = []byte{}
```

Comment:
```suggestion
	newKVs[string(slices.Concat([]byte{0, oidIndex}, commsuffix))] = []byte{}
```
but i'd just concat strings

Reply:
but why? i usually use `slices.Concat` when there are more than two slices otherwise it is useless for me

Reply:
dont insist